### ISSUE TITLE
Add more convenient way to switch between Julia versions

### ISSUE THREAD

**Issue by asampal** [JuliaEditorSupport/julia-vscode#191]
It would be nice to have a more convenient way to switch between installed Julia versions than by changing `julia.executablePath`.

**Comment by davidanthoff** [comment]
Yes, I totally agree, that would be nice. It would also be nice if the extension automatically detected installed julia versions, at least on Mac and Windows that should be fairly straightforward if they are stored in the normal installation locations.
